### PR TITLE
Patch b1 readme reduce (#20)

### PR DESCRIPTION
**Pull request by devrimcavusoglu**
* requirements.txt updated (tqdm version loosen).

* requirements.txt updated. Removed packages that `datasets` depends on.

* Jury moved under __init__.py. black package added to requirements-dev.txt.

* v0.0.5

* requirements.txt updated. Removed packages that `datasets` depends on.

* Jury moved under __init__.py. black package added to requirements-dev.txt.

* v0.0.5

* reduce_fn added. README.md updated.

* Patch b1:
- README.md updated.
- Utility tests added.
- Test scripts added.
- ci.yml updated.
- InputList renamed as Collator.
- setup.cfg added for code style (flake8).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +38,46 @@ Specify metrics you want to use on instantiation.
...
+        def _compute(self, predictions, references, *args, **kwargs):
+            pass
+
+## <div align="center"> Contributing </div>
+
+PRs are welcomed as always :)
+
+### Installation
+
+    git clone https://github.com/obss/pigeons.git
```
Comment (sinanonur) [inline README.md:60]: We dont need pigeons flying over here :)

File: README.md [hunk 2]
```diff
@@ -37,9 +38,46 @@ Specify metrics you want to use on instantiation.
...
+            pass
+
+## <div align="center"> Contributing </div>
+
+PRs are welcomed as always :)
+
+### Installation
+
+    git clone https://github.com/obss/jury.git
+    cd pigeons
```
Comment (sinanonur) [inline README.md:61]: Still pigeons flying around @devrimcavusoglu

Comment (devrimcavusoglu) [reply README.md]: my bad, I've look for entire package now, so it should be ok.

File: tests/run_code_style.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -xeu
+
+if [ $1 = "check" ]; then
```
Comment (fcakyon) [inline tests/run_code_style.sh:5]: maybe we can move these lines to .github/workflows/ci.yml and remove sh file?

File: pyproject.toml [hunk 4]
```diff
@@ -5,6 +5,7 @@ exclude = '''
   /(
```
Comment (fcakyon) [inline pyproject.toml:5]: maybe we can change `line-length = 119` to `line-length = 120` :D

**Review by fcakyon — CHANGES_REQUESTED** [review]

File: tests/run_code_style.sh (hunk 3 above)
Comment (devrimcavusoglu) [reply tests/run_code_style.sh:5]: Just preserving these scripts for local use for developers, but currently converting those scripts to python scripts to preserve platform agnostic structure.

**Review by fcakyon — APPROVED** [review]

**Pull request merged**